src/react-ui-basics/AutocompleteSelect.js

```diff
@@ -215,11 +215,11 @@ class AutocompleteSelect extends React.Component {
                                    selected={selected}
                                    childComponent={childComponent}
                                    childProps={childProps}
-                                   data={data}
+                                   data={[filterValue, ...data]}
```

Comment:
this should be allowed  only with `allowCustom` flag

Reply:
added condition